packages/storybook/stories/va-button-uswds.stories.jsx

```diff
@@ -164,15 +171,14 @@ const TemplateWithForm = ({
 export const Submitted = TemplateWithForm.bind(null);
 Submitted.args = {
   ...defaultArgs,
-  onsub : (e)=>{ 
-    console.log(e.target, "I am submitted!");
-    alert ("form onsubmit method fired!--on form");
+  handleSubmit : (event)=>{ 
+    console.log("Form onSubmit callback fired!", event);
+    alert ("Form onSubmit callback fired!");
```

Comment:
Just some cleanup/clarification of the Storybook page. This is not related to the custom text update.